src/lite-yt-embed.js

```diff
@@ -41,15 +46,13 @@ class LiteYTEmbed extends HTMLElement {
 
         this.style.backgroundImage = `url("${this.posterUrl}")`;
 
-        let playBtn = this.querySelector('.lty-playbtn');
-
         if (!playBtn) {
             playBtn = document.createElement('button');
             playBtn.type = 'button';
             playBtn.classList.add('lty-playbtn');
-            playBtn.title = decodeURIComponent(this.playLabel);
             this.append(playBtn);
         }
+        playBtn.innerHTML = `<span class="lyt-visually-hidden">${decodeURIComponent(this.playLabel)}</span>`;
```

Comment:
i'm gonna do a followup to drop this innerHTML and the insertAdjacentHTML. they give me the heebie jeebies